README.md

```diff
@@ -193,11 +193,24 @@ source ~/.helium.env
 ```bash
 
 ## create the Key Vault
-## do not disable soft-delete in production!
-az keyvault create --enable-soft-delete false -g $He_App_RG -n $He_Name
+az keyvault create -g $He_App_RG -n $He_Name
 
 ```
 
+Key Vault does a soft delete when deleting vaults. If you have gone through this setup already, you could run into errors like "Exist soft deleted vault with the same name.", and "Secret is currently in a deleted but recoverable state ...". You can check if you have deleted vaults and keys with the commands below.
+
+```shell
+
+# list deleted keyvaults that still exist
+az keyvault list-deleted -o table
+
+# list deleted secrets that still exist
+az keyvault secret list-deleted --vault-name $He_Name -o table
+
+```
+
+If you see the Helium vaults or secrets in this state, you can purge or recover the values before moving forward.
```

Comment:
🤔 💭 I thought about adding some example commands here for purging and restoring, but thought it would be going into too much detail. I'm curious what others think. I don't mind adding more examples if others feel it is necessary.

Reply:
I would list the delete / purge commands so that the KV can be fully deleted (so that you can reuse the KV name)